config/endpoints/api_entreprise/13_dgfip_liasses_fiscales.yml

```
@@ -112,4 +112,8 @@
 
 
         {:.fr-highlight.fr-highlight--caution}
-        > La date de déclaration ne vous permet donc pas de distinguer l'imprimé rectificatif de l'imprimé initial car les deux imprimés ont toujours la même date de déclaration. Celle-ci correspond à la date du correctif.
\ No newline at end of file
+        > La date de déclaration ne vous permet donc pas de distinguer l'imprimé rectificatif de l'imprimé initial car les deux imprimés ont toujours la même date de déclaration. Celle-ci correspond à la date du correctif.
+    - q: Pourquoi ais-je parfois des imprimés en double ?
+      a: |+
+        Dans le cas où une société fait partie d'un groupe IS (groupe de société soumis à un régime d'intégration fiscale sur l'Impôt sur les Sociétés), l'entreprise souscrit une déclaration de résultat pour elle-même et une autre en tant que membre du groupe. Dans ce cas, les imprimés peuvent être renvoyés en double.
```

```suggestion
        Dans le cas où une société fait partie d'un groupe IS (c'est-à-dire qu'elle est rattachée à une société mère soumise à l'impôt sur les sociétés), l'entreprise souscrit une déclaration de résultat pour elle-même et une autre en tant que membre du groupe. Dans ce cas, l'API peut renvoyer les imprimés en double : un pour chacune des deux situations.
```

J'aurai reformulé ainsi, mais auparavant, j'aurai peut-être vérifié auprès du FD si ce cas se vérifie à chaque fois : est-ce que tous les SIREN de sociétés ayant une société mère renvoient un double imprimé ?

---

Je dois les recontacter donc je demanderais mais ça serait fort étonnant que ça soit pas le cas

---

Ok dès que possible, dans la doc, j'essaie de mettre quand on est certain d'un comportement pour un périmètre donné. Donc si tu obtiens confirmation, c'est utile

---

```suggestion
        Dans le cas où une société fait partie d'un groupe IS (c'est-à-dire qu'elle est rattachée à une société mère soumise à l'impôt sur les sociétés), l'entreprise souscrit une déclaration de résultat pour elle-même et une autre en tant que membre du groupe. Dans ce cas, l'API peut renvoyer les imprimés en double : un pour chacune des deux situations.
        La déclaration des liasses fiscales étant un acte déclaratif, Il n'est pas possible d'affirmer que le renvoi de deux imprimés sera systématique dans le cas d'une société fille. En outre, une entreprise peut entrer et sortir du périmètre d'un groupe et donc sa situation peut évoluer. 
```